app/serializers/user_serializer.rb

```diff
@@ -43,10 +23,6 @@ def max_subjects
     @model.subject_limit
   end
 
-  def uploaded_subjects_count
```

Comment:
Also needs to be removed from the "include_#{me_only_attribute}?" thing in the private section of this serializer I presume.